### PR TITLE
chore(deploy): add deployment scripts and fix gemspec

### PR DESCRIPTION
**Pull request by JMounier**
- [x] Bump dependencies
- [x] Add deploy and test script
- [x] Update .gemspec

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+[[tool.poetry.source]]
+name = "gitlab"
+url = "https://gitlab.com/api/v4/projects/20917546/packages/pypi/simple"
+
+
+[tool.poetry.dev-dependencies]
+mypy = "^0.790"
+black = "^20.8b0"
+flake8 = "^3.8.3"
```
Comment (tux3) [inline pyproject.toml]: Pas besoin de flake/black/mypy si on ne les run pas, je pense =)

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "tanker-ruby-identity-sdk"
+version = "0.1.0"
+description = "Ruby Identity SDK for Tanker"
+authors = ["Tanker team <tech@tanker.io>"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+tankerci = { version = ">= 2020.12.3123", source = "gitlab" }
```
Comment (maximerety) [inline pyproject.toml]: Nitpick: maybe start by requiring a more recent version when introducing the requirement? E.g.  `2021.6.3413` which has been picked up initially.

**Review by maximerety — CHANGES_REQUESTED** [review]
Nitpick: could you fix the `deployement` typo in the commit title + MR title too?

**Review by tux3 — APPROVED** [review]
LGTM!

**Review by maximerety — APPROVED** [review]

**Pull request merged**